src/djangosaml2_spid/views.py

```diff
@@ -179,69 +156,59 @@ def spid_login(request,
     idps = available_idps(conf)
     if selected_idp is None and len(idps) > 1:
         logger.debug('A discovery process is needed')
-        return render(request, wayf_template, 
-            {
-                'available_idps': idps.items(),
-                'next_url': next_url
-            }
-        )
+        return render(request, wayf_template, {
+            'available_idps': idps.items(),
+            'next_url': next_url
+        })
     else:
         # otherwise is the first one
         _msg = 'Unable to know which IdP to use'
         try:
             selected_idp = selected_idp or list(idps.keys())[0]
-        except TypeError as e: # pragma: no cover
+        except TypeError as e:  # pragma: no cover
             logger.error(f'{_msg}: {e}')
-            return HttpResponseError(_msg)
-        except IndexError as e: # pragma: no cover
+            return HttpResponse(_msg)
```

Comment:
It would be != 200
please use a >400 status code error in the Response

Reply:
The scope of this PR wasn't to patch bugs; I just changed the not existing `HttpResponseError` class with the existing `HttpResponse` to remove the error in the IDE. 😅 

I agree with you though, this should be corrected and I will do it in this PR.

Reply:
@peppelinux Done! 👍🏻
Check it out and resolve the conversation.